### PR TITLE
IBM-Swift/#581 Now able to get port number 

### PR DESCRIPTION
**Pull request by rob-deans**
## Description
Now get port number from `RouterRequest`

## Motivation and Context
Added `request.port` instead of fixing `request.parsedURL.port` as there is no way of getting the port number inside `request.parsedURL` without changing a lot of the code inside `URLParser` and `RouterRequest`

 Has This Been Tested?

## Checklist:

- [ ] I have submitted a [CLA form](https://github.com/IBM-Swift/CLA)
- [ ] If applicable, I have updated the documentation accordingly.
- [ ] If applicable, I have added tests to cover my changes.

### CODE REVIEW

**Comment by rob-deans** [comment]
@shmuelk please can you review this? Thanks

File: README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Let's develop your first Kitura web application!
 
   ```
   $ cd myFirstProject
-  $ swift package init
+  $ swift package init --type executable
```
Comment (akrabat) [inline README.md]: This change is already in master?

Comment (rob-deans) [inline README.md]: I am not sure why that there?

**Comment by rob-deans** [comment]
@shmuelk can you review once more please? thanks

**Pull request merged**